mesheryctl/internal/cli/root/design/error.go

```diff
@@ -22,73 +22,81 @@ import (
+	ErrInvalidDesignFileCode     = "mesheryctl-1038"
+	ErrDesignInvalidNameOrIDCode = "mesheryctl-1039"
+	ErrPatternSourceTypeCode     = "mesheryctl-1121"
+	ErrCopyDataCode              = "mesheryctl-1122"
+	ErrCreateFileCode            = "mesheryctl-1123"
+	ErrRetrieveHomeDirCode       = "mesheryctl-1124"
+	ErrReadFromBodyCode          = "mesheryctl-1125"
+	ErrMarkFlagRequireCode       = "mesheryctl-1126"
 )
 
 const (
-	errPatternMsg = `Usage: mesheryctl pattern import -f [file/url] -s [source-type]
-Example: mesheryctl pattern import -f ./pattern.yml -s "Kubernetes Manifest"`
+	errPatternMsg = `Usage: mesheryctl design import -f [file/url] -s [source-type]
+Example: mesheryctl design import -f ./pattern.yml -s "Kubernetes Manifest"`
 
-	errOnboardMsg = `Usage: mesheryctl pattern onboard -f [filepath] -s [source type]
-Example: mesheryctl pattern onboard -f ./pattern.yml -s "Kubernetes Manifest"
+	errOnboardMsg = `Usage: mesheryctl design onboard -f [filepath] -s [source type]
+Example: mesheryctl design onboard -f ./pattern.yml -s "Kubernetes Manifest"
 Description: Onboard pattern`
 )
 
-func ErrPatternNotFound() error {
-	return errors.New(ErrPatternsNotFoundCode, errors.Fatal, []string{"Pattern Not Found"}, []string{"No Pattern found with the given name or ID"}, []string{"Pattern with the given name or ID is not present"}, []string{"Please check if the given pattern name or ID is present via 'mesheryctl pattern list'"})
+func ErrDesignNotFound() error {
+	return errors.New(ErrDesignsNotFoundCode, errors.Fatal, []string{"Design Not Found"}, []string{"No Design found with the given name or ID"}, []string{"Design with the given name or ID is not present"}, []string{"Please check if the given design name or ID is present via 'mesheryctl design list'"})
 }
 
-func ErrInvalidPatternFile(err error) error {
-	return errors.New(ErrInvalidPatternFileCode, errors.Fatal, []string{err.Error()}, []string{"Pattern appears invalid. Could not parse successfully"}, []string{"Pattern file provided is not valid"}, []string{"Please check that your pattern file is a valid yaml file"})
+func ErrInvalidDesignFile(err error) error {
+	return errors.New(ErrInvalidDesignFileCode, errors.Fatal, []string{err.Error()}, []string{"Design appears invalid. Could not parse successfully"}, []string{"Design file provided is not valid"}, []string{"Please check that your design file is a valid yaml file"})
 }
 
 func ErrPatternInvalidNameOrID(err error) error {
 	return errors.New(
-		ErrPatternInvalidNameOrIDCode,
+		ErrDesignInvalidNameOrIDCode,
 		errors.Alert,
-		[]string{"Unable to fetch Pattern"},
+		[]string{"Unable to fetch Design"},
 		[]string{err.Error()},
-		[]string{"Invalid pattern name or ID"},
-		[]string{"Run `mesheryctl pattern view -a` to view all available patterns."})
+		[]string{"Invalid design name or ID"},
+		[]string{"Run `mesheryctl design view -a` to view all available designs."})
 }
 
 func ErrImportPattern(err error) error {
```

Comment:
```suggestion
func ErrImportDesign(err error) error {
```
Should it be updated ?